pkg/operator/operator.go

```diff
@@ -63,6 +63,7 @@ type Operator struct {
 // +kubebuilder:rbac:groups="",resources=services;endpoints;pods;nodes,verbs=get;list;watch
 // +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterroles;clusterrolebindings,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=networking.k8s.io,resources=ingresses,verbs=get;list;watch
+// +kubebuilder:rbac:groups=route.openshift.io,resources=routes,verbs=get;watch;list
```

Comment:
Maybe it'll come later as PR is still WIP but I don't think that the operator needs any rights on the OpenShift Route.

Reply:
I didn't see the operator code which would need these permissions.
Can you please explain why the operator needs these rights?